content/kapp-controller/docs/latest/packaging.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Packaging (v0.17.0-alpha.1+)
+---
+
+The new alpha release of kapp-controller adds new apis to
+bring common package management workflows to a kubernetes cluster.
+This is done using three new CRs, PackageRepository, Package, and
+InstalledPackage, which are described further in their respective sections.
+As this is still an alpha, we would love any and all feedback regarding these
+apis or any documentation relating to them!
+
+**Disclaimer:** These apis are still very much in an alpha stage, so changes
+will almost certainly be made and no backwards compatibility is guaranteed
+between alpha versions.
+
+## Installation
+
+Just run:
+
+```bash
+$ kapp deploy -a kc -f https://github.com/vmware-tanzu/carvel-kapp-controller/tree/dev-packaging/release.yml
+```
+
+or
+
+```bash
+$ kubectl apply -f https://github.com/vmware-tanzu/carvel-kapp-controller/tree/dev-packaging/release.yml
+```
```

Comment:
We should update these to the locations to develop branch location once merged in for kapp-controller. We still need to pick where the release YAML will go for the alphas though.

Reply:
Yea, for the alpha I left this as kind of a place holder until we were a little more set on the location of the assets. Either just a `release.yml` at the root or in a subdirectory makes the most sense to me. What do you think?

Reply:
I was thinking a folder called alpha_releases with subfolders corresponding to each alpha release (e.g. `alpha_releases/rc-<iteration>/release.yml`. So we can preserve each rc.

Reply:
Yea, I like that, I can update the link